README.md

```diff
@@ -58,82 +57,37 @@ Installation will take several minutes.
 - Run site by using:
 
     ```shell
-    rake serve
+    npm run watch
     ```
-- View the site in your browser at http://localhost:4000
+- View the site in your browser at http://localhost:3000/development/
```

Comment:
If you pull in PR 2702, the `/development/` is not necessary (and actually incorrect)